### ISSUE TITLE
subscriber.py receive requires an args.timeout but it is missing from positional or optional arguments

### ISSUE THREAD

**Issue by phx-srankin** [GoogleCloudPlatform/python-docs-samples#2633]
## In which file did you encounter the issue?
subscriber.py

GoogleCloudPlatform/python-docs-samples/pubsub/cloud-client

### Did you change the file? If so, how?
No

## Describe the issue
I was following the directions here [1] to receive from a subscription I created in the emulator and it fails saying it has no args.timeout. I see no way to provide this or any code that would parse this even if provided.

[1] https://cloud.google.com/pubsub/docs/emulator

python subscriber.py my-project-id receive my-sub
Traceback (most recent call last):
  File "subscriber.py", line 541, in <module>
    receive_messages(args.project_id, args.subscription_name, args.timeout)
AttributeError: 'Namespace' object has no attribute 'timeout'

python subscriber.py my-project-id receive my-sub timeout=5
usage: subscriber.py [-h]
                     project_id
                     {list_in_topic,list_in_project,create,create-push,delete,update,receive,receive-custom-attributes,receive-flow-control,receive-synchronously,receive-synchronously-with-lease,listen_for_errors}
                     ...
subscriber.py: error: unrecognized arguments: timeout=5

**Comment by gguuss** [comment]
Reassigning to Alex who may have more information about what's going on with PubSub.

**Comment by busunkim96** [comment]
It looks like Alex is OOO right now. @anguillanneuf could you take a look?

**Comment by anguillanneuf** [comment]
@phx-srankin Thanks for noticing this! We recently updated all the subscriber samples in GitHub. Adding this line `    receive_parser.add_argument("--timeout", default=None, type=float)` beneath the code here will help. I will work on a PR. 

https://github.com/GoogleCloudPlatform/python-docs-samples/blob/9582dbc9cd828151407c6956f246bebfa4851728/pubsub/cloud-client/subscriber.py#L483-L486

Something else that you might notice when running this code (but won't affect receiving messages) is [`AttributeError: 'NoneType' object has no attribute 'received_messages'`](https://github.com/googleapis/google-cloud-python/issues/9975).  It has been fixed via this [PR](https://github.com/googleapis/google-cloud-python/pull/9982) but the update will only show up in the next release of the library. 

Thanks for your patience!
